zapisy/apps/theses/models.py

```diff
@@ -125,6 +126,12 @@ def is_supporting_advisor_assigned(self, user):
     def is_among_advisors(self, user):
         return self.is_mine(user) or self.is_supporting_advisor_assigned(user)
 
+    def is_user_privileged_for_thesis(self, user):
```

Comment:
Wydaje mi się, że to się rozumie samo przez się.
```suggestion
    def is_user_privileged(self, user):
```

Reply:
Jak najbardziej